pom.xml

```diff
@@ -152,7 +152,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
```

Comment:
we can't do this. the library promises that there are no third-party dependencies. we can't use Guava in production code, only in tests